app/src/main/java/com/github/houseorganizer/houseorganizer/billsharer/Debt.java

```diff
@@ -0,0 +1,41 @@
+package com.github.houseorganizer.houseorganizer.billsharer;
+
+public class Debt {
+
+    private String debtor, creditor;
+    private int amount;
```

Comment:
I think the amount should be floats no ?

Reply:
+1

Reply:
good catch